spec/index.bs

```diff
@@ -9837,7 +9841,7 @@ interface GPUCanvasContext {
 
     1. Let |texture| be |context|.{{GPUCanvasContext/[[currentTexture]]}}.
     1. If any of the following requirements is unmet, return a transparent black image of size
-        |context|.{{GPUCanvasContext/[[size]]}} and stop.
+        |context|.{{GPUCanvasContext/[[textureDescriptor]]}}.{{GPUTextureDescriptor/size}} and stop.
```

Comment:
`[[textureDescriptor]]` may be `null`. Seems like we could simply use the canvas width and height directly here, given that we don't do any adjustment to them when setting the texture descriptor size?